openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -109,6 +110,29 @@
 # .. toggle_tickets: https://openedx.atlassian.net/browse/ARCHBOM-1718
 LOG_REQUEST_USER_CHANGES = getattr(settings, 'LOG_REQUEST_USER_CHANGES', False)
 
+# .. toggle_name: VERIFY_USER_CHANGE_UNCONDITIONAL
+# .. toggle_implementation: SettingToggle
+# .. toggle_default: False
+# .. toggle_description: If False, only check for user mismatch if a session cookie is being set in
+#   the response (the status quo behavior). If True, check for user mismatches regardless of
+#   response cookie status (the desired future behavior). Once this has been enabled for a week or
+#   so, the toggle can be removed in favor of the True setting.
+#   
```

Comment:
pycodestyle complains about this trailing whitespace, but I think it might be needed in order to keep this from being treated as the end of the annotation. :-/